src/main/java/org/apache/sysds/protobuf/SysdsProtos.java

```diff
@@ -0,0 +1,2985 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// source: src/main/resources/protobuf/Frame.proto
+
+package org.apache.sysds.protobuf;
+
+public final class SysdsProtos {
+  private SysdsProtos() {}
```

Comment:
- If you change indentation to Tabs that would be great I don't know if it is possible while it is auto generated.. 
- If you write the Generated in the first line above license then we have the same format as the DML files auto-generated
- Also we have a code-style file in `/dev/docs/CodeStyle_eclipse.xml`, that could be applied again if possible